Modules/Scripted/ShapeAnalysisToolBox/DataImporter.py

```diff
@@ -487,6 +640,124 @@ def getTopologyAndConsistencyString(self, nodeName, inputSegmentName):
     """
     return self.getTopologyString(nodeName, inputSegmentName), self.getConsistencyString(nodeName, inputSegmentName)
 
+  #
+  #FreeSurfer tab functions
+  #
+  def initFreeSurferLUT(self,LUT_path):
+    #import labels LUT
+    numbers=['0','1','2','3','4','5','6','7','8','9']
+    self.freesurfer_lut_dict=dict()
+    with open(LUT_path,'r') as LUT:
+      line = LUT.readline()
+      while line:
+        filtered_line=list(filter(None, line.split(' ')))
+        if len(filtered_line)>0 and filtered_line[0][0] in numbers:
+          color = [int(filtered_line[2])/255.0,int(filtered_line[3])/255.0,int(filtered_line[4])/255.0]
+          self.freesurfer_lut_dict[filtered_line[0]]={'name':filtered_line[1],'color':color}
+        line = LUT.readline()
+
+  def getFreeSurferAvailableSegmentIds(self,template_path):
+    #check available labels for ONE subject, 
```

Comment:
To document python code consider using docstring like here: https://github.com/Kitware/SlicerSALT/blob/8067dc50b0b8cacc603fdffb8c3bea63e606ca71/Modules/Scripted/Home/Home.py#L12-L13